examples/avatar-preview-app/public/data.json

```
@@ -210,5 +210,38 @@
             "name": "07",
             "index": 7
         }
+    ],
+    "animations": [
+        {
+            "name": "Idle",
+            "file": "http://localhost:3000/anim/idle",
+            "idle": true
+        },
+        {
+            "name": "Appear",
+            "file": "http://localhost:3000/anim/spawn_and_wave",
+            "emote": "😎",
+            "appear": true
+        },
+        {
+            "name": "Clap",
+            "file": "http://localhost:3000/anim/clap",
+            "emote": "👏"
+        },
+        {
+            "name": "Wave",
+            "file": "http://localhost:3000/anim/pick_me",
+            "emote": "👋"
+        },
+        {
+            "name": "ThumbsDown",
+            "file": "http://localhost:3000/anim/thumbs_down",
+            "emote": "👎"
+        },
+        {
+            "name": "ThumbsUp",
+            "file": "http://localhost:3000/anim/thumbs_up",
+            "emote": "👍"
+        }
```

(1/2) It would be more convenient for the urls to be complete in the data (as they're likely to be a string as a result of importing from local files and not nice to remove extensions from.
```suggestion
    ],
    "animations": [
        {
            "name": "Idle",
            "file": "http://localhost:3000/anim/idle.glb",
            "idle": true
        },
        {
            "name": "Appear",
            "file": "http://localhost:3000/anim/spawn_and_wave.gbl",
            "emote": "😎",
            "appear": true
        },
        {
            "name": "Clap",
            "file": "http://localhost:3000/anim/clap.glb",
            "emote": "👏"
        },
        {
            "name": "Wave",
            "file": "http://localhost:3000/anim/pick_me.glb",
            "emote": "👋"
        },
        {
            "name": "ThumbsDown",
            "file": "http://localhost:3000/anim/thumbs_down.glb",
            "emote": "👎"
        },
        {
            "name": "ThumbsUp",
            "file": "http://localhost:3000/anim/thumbs_up.glb",
            "emote": "👍"
        }
```